frontend/src/components/@common/HelpPanel/HelpPanel.tsx

```diff
@@ -0,0 +1,21 @@
+import { BiHelpCircle } from "react-icons/bi";
+import styles from "./HelpPanel.module.css";
+
+const HelpPanel = () => {
+  return (
+    <div className={styles["help-panel-container"]}>
+      <BiHelpCircle className={styles["help-panel"]} />
+      <div className={styles["help-modal-container"]}>
+        <ul className={styles["help-modal"]}>
+          <li className={styles["help-text"]}>예제 테스트 성적은 실제 성적과 무관합니다.</li>
+          <li className={styles["help-text"]}>제출완료 후에는 예제 테스트를 실행할 수 없습니다.</li>
+          <li className={styles["help-text"]}>
+            예제 테스트를 실행하지 않아도 제출 된 Pull Request를 기반으로 채점을 진행합니다.
+          </li>
+        </ul>
+      </div>
+    </div>
+  );
+};

```

Comment:
c: 지금은 탭과 ESC로 포커스 컨트롤이 안되네요 . 툴팁 버튼을 button으로 감싸고 ARIA role을 적용하면 접근성이 더 좋아질 것 같아요. 아래는 제가 한번 수정해본 코드인데 참고하셔도 좋을 것 같아요. `role="tooltip"`은 [여기](https://github.com/lezhin/accessibility/blob/master/aria/README.md#tooltip)가 잘 나와있네요.

```suggestion
const HelpPanel = ({ tooltipId }: { tooltipId: string }) => {
  return (
    <div className={styles["help-panel-container"]}>
      <button
        aria-describedby={tooltipId}
        className={styles["help-panel"]}
        onKeyDown={(e) => {
          if (e.key === "Escape") {
            e.currentTarget.blur();
          }
        }}
      >
        <BiHelpCircle />
      </button>
      <div role="tooltip" id={tooltipId} className={styles["help-modal-container"]}>
        <ul className={styles["help-modal"]}>
          <li className={styles["help-text"]}>예제 테스트 성적은 실제 성적과 무관합니다.</li>
          <li className={styles["help-text"]}>제출완료 후에는 예제 테스트를 실행할 수 없습니다.</li>
          <li className={styles["help-text"]}>
            예제 테스트를 실행하지 않아도 제출 된 Pull Request를 기반으로 채점을 진행합니다.
          </li>
        </ul>
      </div>
    </div>
  );
};
```

Reply:
질문이 있습니다.
1. hover 상태인데 esc 컨트롤이 필요할까요??
2. active는 모바일에서 쓰이는데 모바일 ux상 esc 컨트롤이 없기 때문에 괜찮지 않을까요??

role = tooltip은 적용하겠습니다!